### ISSUE TITLE
Text label for system tray icon

### ISSUE THREAD

**Issue by tseykovets** [nkanaev/yarr#229]
YARR's system tray icon does not have a text label. This is an accessibility issue and creates inconvenience for users of assistive technologies such as screen readers.

It would be nice if the icon had a text label and also contained information about unread articles, for example:

* When there are no unread articles: YARR
* When there are unread articles: YARR (15 unreads)

Note: The issue was detected on Windows, but it is most likely typical for builds for all operating systems, for example, for an icon in the Dock panel on macOS.

**Comment by nkanaev** [comment]
Added tooltip to the systray icon: https://github.com/nkanaev/yarr/commit/f06fc1f750a325f2ffff4b6c3f93193432a7a106.
